packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -51,6 +56,34 @@ export async function activate(context: ExtensionContext): Promise<void> {
     clientOptions
   );
 
+  const jsonSchemaConfig:
+    | WorkspaceConfiguration
+    | undefined = workspace.getConfiguration().get("json.schemas");
+  if (
+    process.env["THEIA_PARENT_PID"] !== undefined &&
+    isArray(jsonSchemaConfig)
+  ) {
+    const providedManifestSchemaConfigs = find(jsonSchemaConfig, (_) => {
+      return some(
+        _.fileMatch,
+        (fileMatchEntry) => fileMatchEntry === "manifest.json"
+      );
+    });
+
+    if (providedManifestSchemaConfigs === undefined) {
+      const manifestSchemaConfig = {
+        fileMatch: ["manifest.json"],
+        url:
+          "https://cdn.jsdelivr.net/gh/SAP/ui5-language-assistant/packages/vscode-ui5-language-assistant/resources/manifest-schema/rel-1.19/schema/schema.json",
+      };
+
+      jsonSchemaConfig.push(manifestSchemaConfig);
+      workspace
+        .getConfiguration()
+        .update("json.schemas", jsonSchemaConfig, true);
```

Comment:
why do we update the global settings? could if affect something else we did not mean to do? would it be safer to update the workspace settings?